app/controllers/registers_controller.rb

```diff
@@ -56,9 +57,11 @@ def field_link_resolver(field, field_value, register_slug: @register.slug, white
     resolver = LinkResolver.new(current_register_slug: register_slug, register_whitelist: whitelist)
 
     if field_value.is_a?(Array)
-      field_value.map { |fv| resolver.resolve(field, fv) }.join(', ').html_safe
+      list = field_value.map { |fv| resolver.resolve(field, fv) }.join(', ')
+      BlueCloth.new(list, auto_links: false).to_html
```

Comment:
we should only apply this where we know the field may contain markdown. So we can do something like:
```
    with_links = if field_value.is_a?(Array)
                   field_value.map { |fv| resolver.resolve(field, fv) }.join(', ').html_safe
                 else
                   resolver.resolve(field, field_value)
                 end
    field['datatype'] == 'text' ? BlueCloth.new(with_links, auto_links: false).to_html : with_links
```

Reply:
Done. The linter was complaining about misalignment of `if`, `else`, and `end` so I've used a ternary instead.

Reply:
OK cool, feel free to merge. By the way I don't believe you need `===` in this case for the comparison.